core/state/trie_prefetcher.go

```diff
@@ -241,6 +269,10 @@ type subfetcher struct {
 	used [][]byte            // Tracks the entries used in the end
 
 	accountHash common.Hash
+
+	interrupted  bool
```

Comment:
Do you think any need for this `interrupted` to be atomic?

Reply:
I am not sure, writing a bool var should be atomic, but in multi-core CPU framework, each core does have its own cache, so concurrent read-write may happen, but it won't panic, I am not sure what is the best practice.

Reply:
Yes, boolean or integer value should be used with Atomic for concurrent access, otherwise the result will depend on platform.
But in this case, I deliberately not use atomic to protect, since `interrupted` will not be concurrently accessed, as currently `triePrefetch` API requests, `close()`, `trie()` and `prefetch()` have to be called within a same routine.

Reply:
`interrupted` is removed now, subfetcher's lifecycle is maintained in mainLoop now, no longer needed